docs/tempo/website/configuration/override.md

```diff
@@ -0,0 +1,107 @@
+## Specify overrides to the default settings
+
+To change the default settings for various trace limits, add an `overrides` section. You can:
+
+- Add the `overrides` setting directly in the configuration file.
+- Create a new yaml file containing the `overrides` settings and then reference it from the configuration file.
+
+You can set the `overrides` to apply to all tenants within the cluster, to specific tenants, or globally to the cluster. Per tenant settings override the settings in the main configuration file. Use the main configuration file as a place to add the default values. The per tenant settings can be used to adjust to limit each tenant individually. For more information, refer to the [Override strategies](#override-strategies) section.
+
+### Standard overrides
+
+To set a new trace limit that applies to all tenants of the cluster:
+
+1. Create an `overrides` section at the bottom of the `/tempo/integration/bench/config.yaml` file.
+1. Add the override parameters under this section. For example:
+
+    ```
+    overrides: 
+        max_traces_per_user: 50000
+    ```
+
+A snippet of a `config.yaml` file showing how the `overrides` section is [here](https://github.com/grafana/tempo/blob/a000a0d461221f439f585e7ed55575e7f51a0acd/integration/bench/config.yaml#L39-L40). A list of common override parameters and their description can  be found in [Override parameters](#override-parameters).
+
+Setting a maximum value for the `overrides` setting is a failsafe way to prevent one tenant from negatively impacting others within the cluster. See also, [Distributor is not accepting traces](../../troubleshooting/#problem-4-distributor-is-not-accepting-traces) for more information.
+
+### Tenant-specific overrides
+
+Sometimes you don't want all tenants within the cluster to have the same settings. To add overrides that are specific to an individual tenant:
+
+1. Create an `overrides` section at the bottom of the `config.yaml` file.
+1.  Add an entry `per_tenant_override_config` to point to a separate file named `overrides.yaml` that contains tenant-specific overrides.
+
+    ```
+    overrides:
+        per_tenant_override_config: /conf/overrides.yaml
+    ```
+
+1. Within the `overrides.yaml` file, add the tenant-specific override parameters. For example:
+
+    ```
+    overrides:
+        "<tenant id>":
+            ingestion_max_batch_size: 5000
+            ingestion_rate_limit: 400000
+            max_spans_per_trace: 250000
+            max_traces_per_user: 100000
+    ```
+## Override strategies
+
+The trace limits specified by the various parameters are, by default, applied as tenant-level limits. For example, a max_traces_per_user setting of 10000 means that each tenant within the cluster has a limit of 10000 traces per user. This is known as a `local` strategy in that the specified trace limits are local to each tenant.
```

Comment:
local means "local to the distributor", not local to the tenant.  global means averaged across all distributors by using a distributor ring.

basically with a local ingestion rate of 10000 then each distributor will independently enforce 10000 spans/second in that distributor.  if you set a global rate of 10000 then it will divide 10000 by the number of distributors (say 5) and each distributor will enforce 2000 spans/second.  if you then scale up distributors to 10 then each distributor will enforce 1000 spans/second.